libpkg/pkg_config.c

```diff
@@ -468,6 +469,12 @@ static struct config_entry c[] = {
 		NULL,
 		"List of regex to ignore while autiditing for vulnerabilities",
 	},
+	{
+		PKG_STRING,
+		"COMPRESSION_FORMAT",
+		NULL,
+		"Set the default compression format",
```

Comment:
I think, it should be described as the `compression format for packages creating`, since `pkg` can use all supported formats when reading pkg files.